code/modules/projectiles/projectile/bullettypes.dm

```diff
@@ -118,9 +118,10 @@ There are important things regarding this file:
 /obj/item/projectile/bullet/shotgun
 	name = "slug"
 	icon_state = "slug"
-	damage = 34
-	armor_penetration = 12
+	damage = 64
```

Comment:
Don't forget that the bull can shoot two shots at once. So 642.75(the damage modifier) is 96 before armor.

Reply:
Changed these values since the PR first went up.